app/android/build.gradle

```diff
@@ -3,6 +3,9 @@
 buildscript {
     repositories {
         mavenCentral()
+        maven {
```

Comment:
You don't need to specify URL here. You can simply add an entry ```jcenter()``` like [here](https://github.com/forcedotcom/SalesforceMobileSDK-Android/blob/unstable/build.gradle#L3). You can probably remove ```mavenCentral()``` too, since ```jcenter()``` mirrors from ```mavenCentral()``` too, if I'm not mistaken.